charts/studio/values.yaml

```diff
@@ -49,26 +49,32 @@ global:
 
   scmProviders:
     github:
-      # -- GitHub Enterprise URL
+      # -- GitHub enabled
+      enabled: false
+
+      # -- GitHub URL
       url: ""
-      # -- GitHub Enterprise API URL
+      # -- GitHub API URL
```

Comment:
Keep `GitHub Enterprise` above parameters that should ONLY be set for GIthub Enterprise. maybe even emphasize if it's destructive to set those with github.com. url, apiUrl only I think.